style-media.css

```diff
@@ -0,0 +1,37 @@
+@media (min-width: 992px) {
+    .midle_box {
+        flex-direction: row;
+        margin-left: 20px;
+        margin-right: 20px;
+    }
+    .wrap_box_text_1 {
+        font-size: 1.5rem;
+    }
+    .foter_block_end {
+        flex-direction: column;
+    }
+    .color-icon {
+        font-size: 22px;
+    }
+    .color_for_link {
+        font-size: 1.1rem;
+    }
+}
+
+@media (min-width: 1500px) {
+    .midle {
```

Comment:
Middle what ? it's a bad name just `middle` should be  - middle-block , middle-wrapper or something. Please rename

Reply:
I did